WcaOnRails/app/models/competition.rb

```diff
@@ -391,6 +396,16 @@ def main_event_id=(event_id)
     super(event_id.blank? ? nil : event_id)
   end
 
+  def events_with_rounds
+    includes_associations = [
+      { rounds: [:competition_event] },
+    ]
+    competition_events
+      .includes(includes_associations)
```

Comment:
If you only need the `includes_associations` variable once, you might as well pass it directly:

```ruby
competition_events
  .includes(rounds: [:competition_event])
```

Reply:
done